jwst/jump/jump_step.py

```diff
@@ -15,6 +16,7 @@ class JumpStep(Step):
 
     spec = """
         rejection_threshold = float(default=4.0,min=0) # CR rejection threshold
+        maximum_cores = option('quarter', 'half', 'all', default=None) # max number of processes to create
```

Comment:
Is flake8 going to complain about having only 1 space before the # to start an in-line comment? I think it likes 2.

Reply:
I think this is OK since it is within a string.  I think `ConfigObj` doesn't care.